components/cards.html

```diff
@@ -0,0 +1,122 @@
+                 </a>
+                 </div> 
+                 <div class="nav-top hidden-sm">
+                    <div class="nav-float-right"><a href = "/index.html" class="hover-effect">home</a></div>
+                    <div class="nav-float-right"><a href = "/documentation.html" class="hover-effect">documentation</a></div>
+                    <div class="nav-float-right"><a href = "https://github.com/sneha-pujari/Component-Library" class="hover-effect">github</a></div>
+                </div>
+    
+            </div>
+        </div>
+        </header>  
+            <nav>
+                <h2>Getting started</h2>
+                <ul>
+                  <li><a class="link-style" href="/documentation.html">Introduction</a></li>
+                </ul>
+                <h2>Components</h2>
+                <ul>
+                    <li><a class="link-style" href="avatar.html">Avatar</a></li>
+                    <li><a class="link-style" href="alerts.html">Alerts</a></li>
+                    <li><a class="link-style" href="badges.html">Badges</a></li>
+                    <li><a class="link-style" href="buttons.html">Buttons</a></li>
+                    <li><a class="link-style" href="cards.html">Cards</a></li>
+                    <li><a class="link-style" href="images.html">Images</a></li>
+                    <li><a class="link-style" href="input.html">Input</a></li>
+                    <li><a class="link-style" href="list.html">List</a></li>
+                    <li><a class="link-style" href="typography.html">Typography</a></li>
+                </ul>
+              </nav>
+              <main>
+                <h2 style="font-size:35px">Cards</h2>
+                <section>
+                        <h3>Card Vertical</h3>
+                        <div class="card-container">
+                            <img class="card-img" src="https://bit.ly/2Om0Udn"> 
+                            <h3>Card Heading</h3>
+                            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an un</p>
+                            <button class="primary-button">Add to cart</button>
+                        </div>
+                        <h3>Card Horizontal</h3>
+                        <div class="horizontal-card">
+                            <img class="card-img" src="https://bit.ly/2Om0Udn"> 
+                            <div class="horizontal-card-content">
+                            <h3>Card Heading</h3>
+                            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an un</p>
+                            <button class="primary-button">Add to cart</button>
+                        </div>
+                            </div>
+                    
+                        <h3>Card with shadow</h3>

```

Comment:
The shadow in the card with shadow looks very light, it's hard to observe maybe make it a bit dark.

Reply:
Yes, corrected. Thanks!